build.sh

```diff
@@ -245,6 +245,11 @@ if [ ${BUILD_FE} -eq 1 -o ${BUILD_SPARK_DPP} -eq 1 ]; then
         cp -r -p ${STARROCKS_HOME}/fe/fe-core/target/starrocks-fe.jar ${STARROCKS_OUTPUT}/fe/lib/
         cp -r -p ${STARROCKS_HOME}/webroot/* ${STARROCKS_OUTPUT}/fe/webroot/
         cp -r -p ${STARROCKS_HOME}/fe/spark-dpp/target/spark-dpp-*-jar-with-dependencies.jar ${STARROCKS_OUTPUT}/fe/spark-dpp/
+        # download aliyun hadoop oss jar, this can not be found in maven repository, so we download it manually
+        wget http://cdn-thirdparty.starrocks.com/broker_thirdparty_jars.tar.gz
```

Comment:
I think it is not a good way to download this jar every build time.
I'd like to prepare this dependence when building third-party

Reply:
ok, I have modified the logic to build FE/BE from third-party.

Reply:
the oss related package has been uploaded to CDN, but docker mirror also needs to be updated after this pr merged.  
@imay @wyb